specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/ManagedInstanceCreateMax.json

```diff
@@ -23,7 +23,9 @@
         "storageSizeInGB": 1024,
         "licenseType": "Full",
         "collation": "SQL_Latin1_General_CP1_CI_AS",
-        "dnsZonePartner": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/testinstance"
+        "dnsZonePartner": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/testinstance",
+        "publicDataEndpointEnabled" : "false",
```

Comment:
Don't think the boolean value should be quoted

Reply:
Thanks. I fixed it.